chef/data_bags/crowbar/bc-template-ceph.json

```diff
@@ -6,7 +6,9 @@
       "master": false,
       "disk_mode": "first",
       "config": {
-        "fsid" : ""
+        "fsid" : "",
+        "osds_in_total": 0,
```

Comment:
No there is a check when `osds_in_total` will be different than 0, this value will be taken.